include/deal.II/lac/constraint_matrix.h

```diff
@@ -373,6 +373,9 @@ class ConstraintMatrix : public Subscriptor
    * and columns correspond to the same degrees of freedom, you'd generate
    * several constraint objects, then shift them, and finally merge() them
    * together again.
+   *
+   * If this object is initialized with an IndexSet, also the shifted indices
```

Comment:
"If this object is initialized with an IndexSet, it has to also contain the indices shifted by @p offset."